articles/client-platforms/_includes/_rules-introduction.md

```diff
@@ -1 +1 @@
-Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](/rules).
\ No newline at end of file
+Rules are one of the most powerful features of Auth0. As JavaScript functions that execute each time a user authenticates, Rules serve as middleware to extend the flexibility of your authentication flow. For a detailed description of Rules, see the [full documentation](/rules).
```

Comment:
Yes, I've been indecisive about this, but when referring to Rules as an Auth0 feature, I tend to like capitalizing it. But either way ok with me.